controller/commentsController.js

```diff
@@ -47,3 +48,42 @@ exports.flagComment = async (req, res, next) => {
     next(error);
   }
 };
+
+exports.updateComment = async (req, res, next) => {
+  const comment_id = req.params.commentId;
+  const commentBody = req.body.content;
```

Comment:
This is a suggestion. You don't need to follow it but it would make you code much more easy to understand. You'd have to update where ever else you reference this variable.

```suggestion
  const content = req.body.content;
```